samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -35,22 +35,29 @@ static struct bt_iso_chan_ops iso_ops = {
 	.disconnected	= iso_disconnected,
 };
 
-static struct bt_iso_chan_io_qos iso_tx_qos = {
-	.sdu = sizeof(uint32_t), /* bytes */
-	.rtn = 2,
-	.phy = BT_GAP_LE_PHY_2M,
+static struct bt_iso_chan_io_qos iso_tx_qos[BIS_ISO_CHAN_COUNT] = {
+	{ .sdu = sizeof(uint32_t), /* bytes */
+	  .rtn = 1,
+	  .phy = BT_GAP_LE_PHY_2M, },
+	{ .sdu = sizeof(uint32_t), /* bytes */
+	  .rtn = 1,
+	  .phy = BT_GAP_LE_PHY_2M, },
 };
 
-static struct bt_iso_chan_qos bis_iso_qos = {
-	.tx = &iso_tx_qos,
+static struct bt_iso_chan_qos bis_iso_qos[BIS_ISO_CHAN_COUNT] = {
+	{ .tx = &iso_tx_qos[0], },
+	{ .tx = &iso_tx_qos[1], },
 };
 
-static struct bt_iso_chan bis_iso_chan = {
-	.ops = &iso_ops,
-	.qos = &bis_iso_qos,
+static struct bt_iso_chan bis_iso_chan[BIS_ISO_CHAN_COUNT] = {
+	{ .ops = &iso_ops, .qos = &bis_iso_qos[0], },
+	{ .ops = &iso_ops, .qos = &bis_iso_qos[1], },
 };
```

Comment:
It is not necessary to have separate IO QoS nor QoS structs for each stream; they can use the same (and in fact for broadcast they shall use the same values)

Reply:
Done